blackbox/docs/sql/reference/set.txt

```diff
@@ -5,48 +5,61 @@
 SET / RESET
 ===========
 
-Alter and restore global cluster setting values at runtime.
-For a list of the possible settings see
-:ref:`conf-cluster-settings`. Only settings documented with
+Change and restore settings at runtime. To get an overview of available Crate
+settings, see :ref:`configuration`. Only settings documented with
 *Runtime:*  ``yes`` can be changed.
 
-
 Synopsis
 ========
 
 ::
 
-    SET GLOBAL [ PERSISTENT | TRANSIENT ] setting_ident [ = | TO ] value [ , ... ]
+    SET [ SESSION | LOCAL ] setting_ident { = | TO } { value | ident | DEFAULT }
```

Comment:
i would do: SET [ SESSION | LOCAL ] setting_ident { = | TO } { value | DEFAULT }
and describe value as they do in the postgres docs.

the description of value is currently missing and needs to be added. i would use the name value for both synopsises but would describe in depth what it is (it is an expression in case of global btw)